platform/ios/Info.plist Keys.md

```diff
@@ -0,0 +1,15 @@
+# Info.plist Keys
+
+The SDK supports several custom `Info.plist` keys in your application in order to configure various settings. 
+
+## MGLMapboxAccessToken
+
+Use this key to set your Mapbox API [access token](https://www.mapbox.com/help/define-access-token/). This key is required for all map use. 
+
+## MGLMapboxAPIBaseURL
+
+Use this key if you need to customize the API base URL used throughout the SDK. If unset, the default Mapbox API is used. 
```

Comment:
An example would be nice (even the default value). Otherwise, the developer has no idea whether to include the scheme, a trailing slash, etc.